### PR TITLE
QA

### PR DESCRIPTION
**Pull request by VinhH2402**
edit className component-specific CSS 
add upload Image component
add function click to thumbnail to expand photo
have add click tracker but not finish yet

### CODE REVIEW

File: client/src/components/QA/AddingForm.jsx [hunk 1]
```diff
@@ -107,7 +117,7 @@ class AddingForm extends React.Component {
               </tr>
               <tr>
                 <td></td>
-                <td className='warning_text'>For privacy reasons, do not use your full name or email address</td>
+                <td className='QA_warning_text'>For privacy reasons, do not use your full name or email address</td>
```
Comment (aleksandar-cakic) [inline client/src/components/QA/AddingForm.jsx:120]: I like that you're using QA abbreviation before the class name, however, I believe we are to use train-case lowercase for CSS styling. Please see this link: https://github.com/rpp29-fec-gouda/atelier/blob/main/docs/web-style-guide.md

File: client/src/components/QA/AddingForm.jsx [hunk 2]
```diff
@@ -127,21 +137,22 @@ class AddingForm extends React.Component {
                   </input>
                 </td>
               </tr>
+      
               <tr>
                 <td></td>
-                <td style={{ color: 'red' }}>{this.state.requires.email}</td>
+                <td className='QA_warning_text'>For authentication reasons, you will not be emailed</td>
```
Comment (aleksandar-cakic) [inline client/src/components/QA/AddingForm.jsx:143]: Same here with regards to the train-case styling

File: client/src/components/QA/Question.jsx [hunk 3]
```diff
@@ -13,7 +14,9 @@ class Question extends React.Component {
     };
   }
 
-  addAnswerClicked(id) {
+  addAnswerClicked(e) {
+    ClickedTracker('add Answer');
```
Comment (aleksandar-cakic) [inline client/src/components/QA/Question.jsx:18]: How does click tracker work in this instance?

File: client/src/components/css/QA.css [hunk 4]
```diff
@@ -12,42 +12,49 @@ td {
...
 }
-.more_question {
+.QA_more_question {
   width: 250px
 }
 
-.question_body{
+.QA_question_body{
   font-weight: bold;
+  font-size: 20px;
```
Comment (aleksandar-cakic) [inline client/src/components/css/QA.css:39]: Try not to use absolute values for CSS. It's better to use relative measurements such as 'em' (for font)

**Review by aleksandar-cakic — APPROVED** [review]
I like your code a lot, it's simple and clean to read and understand. I noticed you are not using the train-case style for CSS. Please consider switching to it for class and id naming. Otherwise great work!

File: client/src/components/QA/Question.jsx (hunk 3 above)
Comment (VinhH2402) [reply client/src/components/QA/Question.jsx:18]: It's not finish yet. I still thinking how I implement to my widget.

File: client/src/components/css/QA.css (hunk 4 above)
Comment (VinhH2402) [reply client/src/components/css/QA.css:39]: I'm confused when to use em or px, I will edit CSS styling and make new PR

**Pull request merged**